src/internal_modules/roc_pipeline/sender_sink.h

```diff
@@ -109,6 +109,9 @@ class SenderSink : public sndio::ISink, public core::NonCopyable<> {
     //! Write frame.
     virtual ROC_ATTR_NODISCARD status::StatusCode write(audio::Frame& frame);
 
+    //! Flush buffered samples to disk
+    virtual ROC_ATTR_NODISCARD status::StatusCode flush();
```

Comment:
Here and in other headers: it's not necessarily flushing *to disk* because e.g. there are pipeline sinks that will later implement flushing to network queue.

We can reword it like:

> Flush buffered data, if any.

Reply:
Done